src/Microsoft.Fx.Portability.MetadataReader/DependencyFinderEngineHelper.cs

```diff
@@ -73,27 +71,25 @@ public void ComputeData()
             }
 
             // Get the assembly info of System.Object and set it as assembly info for primitives
-            if (_assemblyFilter is DotNetFrameworkFilter)
+            var systemObjectMemberDependency = MemberDependency.FirstOrDefault(t => string.Equals(t.MemberDocId, "T:System.Object", StringComparison.Ordinal) && _assemblyFilter.IsFrameworkAssembly(t.DefinedInAssemblyIdentity));
```

Comment:
Hmm, that is true. If they created their own framework, would they want to check for API compatibility with our .NET framework APIs? We use [DotNetFrameworkFilter](https://github.com/Microsoft/dotnet-apiport/blob/a5988f7c0cd01ce31b83ea7bdee39b8abf4d4c38/src/Microsoft.Fx.Portability/Analyzer/DotNetFrameworkFilter.cs) to calculate which APIs to send up. I would opt to keep for the consistency. but I am OK either way.